flutter/pubspec.yaml

```diff
@@ -109,6 +109,7 @@ dependencies:
   xterm: 4.0.0
   sqflite: 2.2.0
   google_fonts: ^6.2.1
+  vector_math: ^2.2.0
```

Comment:
Build error on my machine (flutter 3.24.5, the version of github CI), maybe you should downgrade to `vector_math: 2.1.4`.

```
Note: vector_math is pinned to version 2.1.4 by flutter_localizations from the flutter SDK.
See https://dart.dev/go/sdk-version-pinning for details.


Because flutter_hbb depends on flutter_localizations from sdk which depends on
  vector_math 2.1.4, vector_math 2.1.4 is required.
So, because flutter_hbb depends on vector_math ^2.2.0, version solving failed.
```

Reply:
Ahh, I am indeed running a newer Flutter SDK and have a handful of local updates to accomodate that.